docs/Advanced-Usage/Debugging/Debugging-Hardware-Using-ILA.rst

```diff
@@ -40,6 +40,19 @@ themselves (eg. in Chisel3.util.Queue).
 Note: In case the module with the annotated signal is instantiated multiple times,
 all instatiations of the annotated signal will be wired to the ILA.
 
+Setting a ILA Depth
+-------------------
+
+The ILA depth parameter specifies the duration in cycles to capture annotated signals
+after a trigger. Increasing this parameter may ease debugging, but will also increase
+FPGA resource utilization. The default depth is 1024 cycles. The desired depth can be
+configured much like the desired HostFrequency by appending a mixin to the 
```

Comment:
I would like @davidbiancolin 's opinion here if the platform config suffix is the correct way to go here

Reply:
Yeah that's perfectly fine, though maybe we'd want a more descriptive name, even just ILADepth<NUM> in case there are more of these in the future. Such as the number of probe triggers.